package.json

```diff
@@ -303,7 +303,7 @@
     "url-search-params": "^0.10.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^8.17.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#55649f9ee704a28c43cbef8d3902d919b07284ca",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#b8a9a2c54f5149dca079099a037ab4839bda7123",
```

Comment:
@kevinmirc - tagging you because this is the most recent commit hash for master of vets-json-schema, which I noticed you made for the 10-10cg. If you don't want these updates in vets-website yet, I can use the one preceding this one.

Reply:
Thanks for the notice. This may affect @LinnJS 's work, but he approved the PR for this commit, so if it breaks anything he should be expecting it.